src/api/main.py

```diff
@@ -69,8 +67,6 @@
     ("/domain/<domain_id>/deployed/", DomainDeployedCheckView),
     ("/domain/<domain_id>/generate/", DomainGenerateView),
     ("/domain/<domain_id>/launch/", DomainLaunchView),
-    ("/domain/<domain_id>/records/", DomainRecordView),
```

Comment:
Is there a reason this was removed? This will break a lot of functionalities on the UI.

Reply:
Weird, not sure why that got removed. I'll add it back